src/main/java/com/github/fishio/control/SinglePlayerController.java

```diff
@@ -172,18 +191,41 @@ public void onMute(ActionEvent event) {
 		
 		AudioEngine.getInstance().toggleMuteState();
 	}
-
+	
+	/**
+	 * Shows a fade-in and fade-out of a pop-up image when an achievement is
+	 * obtained.
+	 */
+	public static void showAchievePopup() {
+		System.out.println("showAchieve reached");
+		int in = 2000;
+		int out = 1000;
+		int duration = 15000;
+		achievePopup.setVisible(true);
+		// imagepop.setImage(popupimage);
```

Comment:
I decided to wait with inserting a picture until the rest works.

<organisation,feature